index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
+            My Recent Works
+        </h1>
+
+    </section>
+
+    <section class="section-bg">
+        <div class="bg">
+        </div>
+    </section>
+
+    <section class="multi-post">
+        <h1 class="multipost-content">
+            Multi-Post Stories
+        </h1>
+    </section>
+
+    <div class="paragraph-box">
+        <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+            <li class="btn ">CSS</li>
+            <li class="btn ">HTML</li>
+            <li class="btn ">Bootstrap</li>
+            <li class="btn ">Ruby</li>
+        </ul>
+        
+        <button class="btn-pink">See Project</button>
+        </div>
+
+<section class="content-on-image-section">
+
+
+    <section class="background-image">
+        <section class="image-content">
+            <div class="image-cont-holder">
+            <h1 class="image-h1">
+                Profesional Art Printing Data
+            </h1>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+
+            <ul class="languages-2">
+              <li class="btn-lang ">HTML</li>
+              <li class="btn-lang ">Bootstrap</li>
+              <li class="btn-lang ">Ruby</li>
+          </ul>
+<div class="button-see-proj">
+    <button class="see-project">
+        See Project
+    </button>
+</div>
+        </div>
+        </section>
+    </section>
+
+    <section class="background-image">
+      <section class="image-content">
+          <div class="image-cont-holder">
+          <h1 class="image-h1">
+              Profesional Art Printing Data
+          </h1>
+          <p class="para">
```

Comment:
- [ ] Kindly change the h1 to a lower hx tag because following semantic HTML on one page it should be only one h1 and it should be in the beginning. Keep rocking💪

Reply:
Okay, done!